src/remote-config/remote-config-api.ts

```diff
@@ -401,6 +400,17 @@ export interface ServerTemplate {
    * project's {@link ServerTemplate}.
    */
   load(): Promise<void>;
+
+  /**
+   * Sets and caches a {@link ServerTemplateData} or a JSON string representing
+   * the server template
+   */
+  set(template: ServerTemplateDataType): void;
```

Comment:
Just curious (because I think either is fine) so we need a separate type for this or should this just be `ServerTemplateData | string` here?

Reply:
I think either works, I originally had it as `ServerTemplateData | string` but we are defining it in quite a few places across multiple files, so I moved it to a type 😃